c/json.c

```diff
@@ -47,6 +47,8 @@
 #include "unixfile.h"
 #include "json.h"
 #include "charsets.h"
+#include "logging.h"
```

Comment:
for this entire file: i'm not sure if zowelog is ok to use here or not. is this a high-enough level library that it is ok?

Reply:
I agree with Sean. We don't need zowelog in this file. We need to look at other json libraries to see how errors are reported.